.gitignore

```diff
@@ -11,6 +11,10 @@ pids
 *.seed
 *.pid.lock
 
+<<<<<<< HEAD
```

Comment:
Definitely delete `<<<< HEAD`, etc. before it's merged.

Reply:
Done